build_projects/dotnet-cli-build/build.sh

```diff
@@ -41,12 +41,11 @@ while [[ $# > 0 ]]; do
             echo ""
             echo "Options:"
             echo "  --configuration <CONFIGURATION>     Build the specified Configuration (Debug or Release, default: Debug)"
-            echo "  --targets <TARGETS...>              Comma separated build targets to run (Init, Compile, Publish, etc.; Default is a full build and publish)"
-            echo "  --nopackage                         Skip packaging targets"
             echo "  --skip-prereqs                      Skip checks for pre-reqs in dotnet_install"
+            echo "  --nopackage                         Skip packaging targets"
             echo "  --docker <IMAGENAME>                Build in Docker using the Dockerfile located in scripts/docker/IMAGENAME"
             echo "  --help                              Display this help message"
-            echo "  <TARGETS...>                        The build targets to run (Init, Compile, Publish, etc.; Default is a full build and publish)"
+            echo "  --targets <TARGETS...>              Comma separated build targets to run (Init, Compile, Publish, etc.; Default is a full build and publish)"
```

Comment:
I think the `--targets` option is important enough, that it should be up at the top where it was.

Removing the "  <TARGETS...> " line is a good change though, since that doesn't work.  Thanks for the sharp eye!